sentry/src/main/java/io/sentry/Baggage.java

```diff
@@ -141,6 +141,7 @@ public static Baggage fromEvent(
     // we don't persist sample rate
     baggage.setSampleRate(null);
     baggage.setSampled(null);
+    // TODO: add replay_id later
```

Comment:
still relevant?

Reply:
yep, will add it when implementing support for ANR (v2) replays, we'd have to serialize replay_id to disk for that and get it from there